ddl-introduction.md

```diff
@@ -178,6 +178,14 @@ When TiDB is adding an index, the phase of backfilling data will cause read and
 
     If a completed DDL task is canceled, you can see the `DDL Job:90 not found` error in the `RESULT` column, which means that the task has been removed from the DDL waiting queue.
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`: Used to pause the DDL tasks that are being executed. After the command is executed, the SQL statement that executes the DDL task is displayed as being executed, and the background task is paused.
+
+    You can pause only DDL tasks that are in progress or still in queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.
```

Comment:
```suggestion
    You can pause only DDL tasks that are in progress or still in the queue. Otherwise, the `Job 3 can't be paused now` error is shown in the `RESULT` column.
```